app/components/modals/cropper-modal.js

```diff
@@ -32,7 +32,7 @@ export default ModalBase.extend({
       this.onVisible();
     },
     cropImage() {
-      this.$('img').croppie('result', 'base64', 'original', 'jpeg').then(result => {
+      this.$('img').croppie('result', { type: 'base64', size: 'original', quality: 1 }).then(result => {
```

Comment:
@maze-runnar Why did you remove the `jpeg` ?

Reply:
i think it's not needed.and it was working fine so i removed it .

Reply:
Please restore it.

Reply:
ok